articles/storage/common/storage-import-export-service-faq.md

```diff
@@ -82,9 +82,18 @@ You can ship drives to the Azure datacenter using any known carrier like FedEx,
 - A FedEx account number in the US and EU, or
 - A DHL account number in the Asia and Australia regions.
 
+Note: India Datacenters require a Declaration letter on customer’s letterhead for returning the drives (Delivery Challan). You must also book the collection of the disks with your selected carrier and share the pickup details with the Datacenter in order to arrange the required entry pass. 
+
 ### Are there any restrictions with shipping my drive internationally?
 Please note that the physical media that you are shipping may need to cross international borders. You are responsible for ensuring that your physical media and data are imported and/or exported in accordance with the applicable laws. Before shipping the physical media, check with your advisors to verify that your media and data can legally be shipped to the identified data center. This will help to ensure that it reaches Microsoft in a timely manner.
 
+### Are there any special requirements for delivering my disks to a DC?
```

Comment:
DC is not an approved acronym for datacenter. Change to: ### Are there any special requirements for delivering my disks to a datacenter?